bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java

```diff
@@ -594,4 +594,11 @@ static Field adjustField(Field field, @Nullable Field existingField) {
     // no adjustment
     return field;
   }
+
+  public static String prepareQueryForLog(String query, int maxLength) {
+    String noNewLinesQuery = query.replace("\n", "");
```

Comment:
A suggestion is to replace "\n" with "\\n". This makes it more readable if there are comments.

Reply:
Done